apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Running TypeScript with a Node.js Itself
```

Comment:
I think that a better title would be `Running TypeScript with Node.js`

Reply:
not really fan of this because when you transpile and next you run it you use node.js so we can use term natively

Reply:
But when you transpile TypeScript, you are not running TypeScript, you are running JavaScript. When you say run TypeScript with Node.js, the meaning is to directly run the TypeScript code with Node.js, otherwise you would have to explicitly mention the transpilation step.